lib/metasploit/framework/data_service/proxy/credential_data_proxy.rb

```diff
@@ -36,10 +37,11 @@ def create_cracked_credential(opts)
   def create_credential_and_login(opts)
     begin
       self.data_service_operation do |data_service|
+        opts = opts.clone
         core = data_service.create_credential(opts)
         opts[:core] = core
         login = data_service.create_credential_login(opts)
```

Comment:
Would it make sense to just delete the `login =` and the next line altogether? That makes it more clear this is just a passthrough. Are there other places where this is wrong?